### DISCUSSION TITLE
Disable the livewire console commands

### DISCUSSION THREAD

**Discussion by murrant** [livewire/livewire#1729] [4. General]
Is there anyway to disable the Livewire Artisan commands?

**Comment by mikebronner** [comment]
You could exclude the default service provider from being loaded, and instead make your own that doesn't load the commands. It would basically be copy/paste, and just leaving out the command-loading: https://github.com/livewire/livewire/blob/e69d9d2bb28eeb55278f191e06963cca4f6cb643/src/LivewireServiceProvider.php#L62
There might be a way to remove commands individually from the IOC, but you'd have to play around with that.

**Reply by murrant** [reply]
That is true, although I hate doing that.  Thanks :)
